utils/src/field_helpers.rs

```diff
@@ -86,6 +87,14 @@ impl<F: Field> FieldHelpers<F> for F {
     }
 }
 
+pub fn i32_to_field<F: From<u64> + Neg<Output = F>>(i: i32) -> F {
```

Comment:
Very useful! I have some of these for Cairo. Nonetheless, I think you can do it with intermediate `u32` instead. Meaning, `i32` has one leading bit for positive/negative, and then 31 bits of actual digits [-2147483648..2147483647], but `u32` has 32 whole bits for natural numbers [0..4294967295]. Nonetheless, I guess that the impact on efficiency is minimal.